fe/fe-core/src/main/java/org/apache/doris/clone/BackendLoadStatistic.java

```diff
@@ -187,7 +187,7 @@ public void init() throws LoadBalanceException {
 
             RootPathLoadStatistic pathStatistic = new RootPathLoadStatistic(beId, diskInfo.getRootPath(),
                     diskInfo.getPathHash(), diskInfo.getStorageMedium(),
-                    diskInfo.getTotalCapacityB(), diskInfo.getDataUsedCapacityB(), diskInfo.getState());
+                    diskInfo.getTotalCapacityB(), diskInfo.getDiskUsedCapacityB(), diskInfo.getState());
```

Comment:
your means that it should statistic the usage of disk instead of the data?

Reply:
Yes, and it seems that `TDisk.getDataUsedCapacity()` only counts all tablets, which means it ignores trash, meta and log